SYNTAX.md

```diff
@@ -93,6 +94,15 @@ stages {
         }
     }
 }
+
+stages {
+    stage('parallel-stage') {
+        parallel([
+            firstBlock: 'echo "First block of the parallel"',
```

Comment:
Somehow as a followup of Michael above, do those blocks really take a string, of shouldn't it be a closure?